spec/virtual_attributes_spec.rb

```diff
@@ -816,17 +816,26 @@ def lc
   end
 
   describe ".order" do
-    it "supports virtual attributes symbol" do
+    it "orders by virtual attributes symbol" do
       Author.order(:nick_or_name).first # fails
     end
 
-    it "supports virtual attributes hash" do
+    it "orders by virtual attributes hash" do
       Author.order(:nick_or_name => "ASC").first # fails
     end
 
-    it "supports virtual attributes arel" do
+    it "orders by virtual attributes arel" do
       Author.order(Author.arel_attribute(:nick_or_name)).first
     end
+
+    it "orders by virtual attributes inline arel" do
+      Author.order(Author.arel_attribute(:nick_or_name).asc).first
+    end
+
+    it "orders by virtual attributes inline arel" do
+      asc_node = Arel::Nodes::Descending.new(Author.arel_attribute(:nick_or_name))
+      Author.order(asc_node).first
```

Comment:
So none of these tests have actual assertions?  That seems... not good.

Reply:
I also see some of these with the comment of `# fails`... does that mean the specs don't pass?

Reply:
FYI:  This don't have to be addressed in this PR, since that isn't the intent, just curious how these new tests should related to the other ones.  Obviously these would raise an error if they were invalid, but it probably also makes sense that they are at least returning data that we expect them to as well (again, not a requirement, just my 2cents)